controller/konnect/ops/ops_kongroute.go

```diff
@@ -140,8 +140,11 @@ func kongRouteToSDKRouteInput(
 		Sources:                 route.Spec.KongRouteAPISpec.Sources,
 		StripPath:               route.Spec.KongRouteAPISpec.StripPath,
 		Tags:                    tags,
-		Service: &sdkkonnectcomp.RouteService{
+	}
+	if route.Status.Konnect.ServiceID != "" {
```

Comment:
```suggestion
	if route.Status.Konnect != nil && route.status.Konnect.ServiceID != "" {
```
Just to make sure we do not hit panic.